### ISSUE TITLE
Running picoquicvpn

### ISSUE THREAD

**Issue by LeonardWalter** [p-quic/pquic#2]
`FATAL ERROR: no protocol operation with id get_message_socket and hash 1748423259152196505`
I always get this error when I'm trying to get the `./picoquicvpn` to work.
I am using Debian 9 with Kernel 4.9.0.7
I followed the guide on [pquic.org](pquic.org) -> `./picoquicdemo` and the fec plugin works without any problem.

**Comment by mpiraux** [comment]
The VPN is a bit more tricky to run than the base demo client/server. It requires two tunnels interfaces and some routes set up to redirect traffic to the VPN. There is a mininet topology in the benchmarks folder that emulate this.

The VPN also requires the datagram plugin to be injected into PQUIC to work. But you have to do it yourself in the arguments, i.e adding -P plugins/datagram/datagram.plugin. Make sure you compiled the plugin using the Makefile at the root of the plugin directory.

**Comment by Winters123** [comment]
HI,
When I `make` pquic, there is an error reported: it said IPV6_DONTFRAG is not defined. Then I modified all this macro as 0. It compiled successfully. But when I ran `./picoquicdemo` . The terminal shows `Starting PicoQUIC server on port 4443, server name = ::, just_once = 0, hrr= 0, 0 local plugins and 0 both plugins
Server exit, ret = -1
Server exit with code = -1
`
I used CentOS 7 as the environment. How should I proceed?

Thanks and regards,
Xiangrui
